js/init.js

```diff
@@ -1,32 +1,48 @@
 function syncState() {
-  chrome.storage.sync.get('classList', function(result) {
+  let board_id = location.pathname.split('/')[2];
```

Comment:
Could you rename board_id to boardId, I think all the other var names here are using camel case.